recipe/meta.yaml

```diff
@@ -49,10 +49,8 @@ about:
     large data in particular and has specific optimizations for numpy arrays."
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE.txt
   license_url: https://github.com/joblib/joblib/blob/{{ version }}/LICENSE.txt
   doc_url: https://joblib.readthedocs.io/en/latest/
```

Comment:
```suggestion
  doc_url: https://joblib.readthedocs.io/
```

Reply:
Maybe it's best to change these urls for those whose native language is not English.

Reply:
Maybe use the same url for the `home` url as well.

`home: https://joblib.readthedocs.io/`

Reply:
Makes sense! Updated. Thanks!